project.py

```diff
@@ -23,10 +23,48 @@ def __init__(self, name, birthdate):
 # Printing value of now.
 print("Time now at greenwich meridian is:", current_time)
 
+#Prints the number of days between now and the new year 
+
 f_date = date(current_time.year,current_time.month,current_time.day)
 l_date = date(2024, 1, 1)
 delta = l_date - f_date
 print(delta.days)
 
+# Creating an array of birthdays
+birthdays = [franks_birthday, lucies_birthday, johns_birthday]
+
+#Printing the next birthday
+def get_next_birthday(birthdays):
+    today = datetime.date.today()
+    next_birthday = None
+    for person in birthdays:
+        # Check if the birthday for this year has already passed, if yes, use the next year's birthday
+        this_year_birthday = date(today.year, person.birthdate.month, person.birthdate.day)
+        if this_year_birthday < today:
+            this_year_birthday = date(today.year + 1, person.birthdate.month, person.birthdate.day)
+        
+        if next_birthday is None or this_year_birthday < next_birthday:
+            next_birthday = this_year_birthday
+    
+    return next_birthday 
+
+#Time till next birthday 
+
+next_birthday = get_next_birthday(birthdays)
+print("Next birthday:", next_birthday)
+
+def time_till_next_birthday(next_birthday):
+    today = datetime.date.today()
+    time_till_birthday = next_birthday - today
+    return time_till_birthday
+
+next_birthday = get_next_birthday(birthdays)
+print("Next birthday:", next_birthday)
+
+time_till_birthday = time_till_next_birthday(next_birthday)
+print("Time till next birthday:", time_till_birthday)
```

Comment:
When you tested this, unless you tested it a 00:00:00 (i.e. midnight) I'd expect some time component telling us exactly how long till midnight of the next birthday. Maybe we need to look at this today?

Reply:
Will fix as part of the following PR